filebeat/module/kibana/audit/test/test-audit-711.log

```diff
@@ -1,4 +1,4 @@
-{"@timestamp":"2020-12-09T11:57:34.870-05:00","message":"User is requesting [/foo/spaces/enter] endpoint","log":{"level":"INFO","logger":"plugins.security.audit.ecs"},"process":{"pid":20699},"ecs":{"version":"1.6.0"},"event":{"action":"http_request","category":"web","outcome":"unknown"},"http":{"request":{"method":"get"}},"url":{"domain":"0.0.0.0","path":"/foo/spaces/enter","port":5603,"scheme":"https:"},"user":{"name":"elastic","roles":["superuser"]},"kibana":{"space_id":"default"},"trace":{"id":"71a7d4d1-e9ba-474c-a844-9d9c1dc11ba5"}}
```

Comment:
This test should verify that we can ingest logs matching the format produced by `7.11`. It looks like we're changing this to conform to the new >=`7.13` format, which isn't the intent of this test. I think we should have another set of tests (`test-audit-713`) to make sure that this pipeline can ingest both formats.

Reply:
oops, thanks for pointing this out - I hadn't noticed the filename.